src/c_interface.rs

```diff
@@ -26,7 +26,7 @@ pub extern "C" fn sysinfo_init() -> CSystem {
 pub extern "C" fn sysinfo_destroy(system: CSystem) {
     assert!(!system.is_null());
     unsafe {
-        Box::from_raw(system as *mut System);
+        drop(Box::from_raw(system as *mut System));
```

Comment:
Why this change?

Reply:
Ah because of the warning, got it.

Reply:
The version of rustc I have warns on just abandoning a Box:

```
warning: unused return value of `Box::<T>::from_raw` that must be used
  --> src/c_interface.rs:29:9
   |
29 |         Box::from_raw(system as *mut System);
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_must_use)]` on by default
   = note: call `drop(from_raw(ptr))` if you intend to drop the `Box`
```